Update set-up-content-understanding.md (#4075)
MicrosoftDocs/microsoft-365-docs: microsoft-365/contentunderstanding/set-up-content-understanding.md

> 2. Select the users that you want to license, and click **Manage product licenses**.

Reviewer: ```suggestion
2. Select the users that you want to license, and choose **Manage product licenses**.
```

> 3. Click on **Apps** drop down.

Reviewer: ```suggestion
3. Choose **Apps** from the drop-down menu.
```

> 4. SelectSshow apps for  **SharePoint Syntex**. Under **Apps**, make sure **Common Data Service for SharePoint Syntex**, **SharePoint Syntex**, and **SharePoint Syntex - SPO type** are all selected.

Reviewer: ```suggestion
4. Select **Show apps for  SharePoint Syntex**. Under **Apps**, make sure **Common Data Service for SharePoint Syntex**, **SharePoint Syntex**, and **SharePoint Syntex - SPO type** are all selected.
```
